tests/adapters/wallet_eth_txns/test_adapter.py

```diff
@@ -1,36 +0,0 @@
-import pytest
```

Comment:
This file was renamed and also re-written to use the fake implementation of `EthClient` to make unit tests unit tests.